plugins/crd/plugin_impl_crd.go

```diff
@@ -227,16 +226,7 @@ func createCRD(plugin *Plugin, FullName, Group, Version, Plural, Name string) er
 			Validation: validation,
 		},
 	}
-
 	_, cserr := plugin.apiclientset.ApiextensionsV1beta1().CustomResourceDefinitions().Create(crd)
-	if cserr != nil && apierrors.IsAlreadyExists(cserr) {
```

Comment:
Why remove these printouts?

Reply:
crdCreate function should not print anything but just return the result, by doing fancy printing using `plugin` logger method, breaks unit testing as it is really hard to initialize logger.